source/default/webpack.config.js

```diff
@@ -31,27 +28,17 @@ module.exports = {
           },
         ],
       },
-      {
-        test: /\.svg$/,
-        loader: 'file-loader',
-        options: {
-          name: '[path][name].[ext]',
-          outputPath: ASSETS_ATOMIC_FOLDER,
-          context: path.resolve(__dirname),
-          emit: true,
-        },
-      },
     ],
   },
   plugins: [
     // Sprite system options
     new SVGSpritemapPlugin(
-      path.resolve(namespaces.atoms, 'svg/icons/**/*.svg'),
+      path.resolve(namespaces.static, 'icons/svg/**/*.svg'),
       {
         output: {
           filename: 'images/spritemap.svg',
           svg4everybody: true,
-          svgo: true,
+          svgo: false,
```

Comment:
Why disable this?

Reply:
@illepic Because we know we want to run these manually based on our own SVGO configuration. We've already cleaned these by the time this plugin is involved and we don't want to remove currentColor.